py/strato/common/log/log2text.py

```diff
@@ -235,12 +235,12 @@ def updateConfFile(field, value):
     import argparse
     parser = argparse.ArgumentParser()
     parser.add_argument("logFiles", metavar='logFile', nargs='*', help='logfiles to read or - for stdin')
-    parser.add_argument("--noDebug", action='store_true', help='filter out debug messages')
-    parser.add_argument("--relativeTime", action='store_true', help='print relative time, not absolute')
-    parser.add_argument("--noColors", action='store_true', help='force monochromatic output even on a TTY')
-    parser.add_argument(
+    parser.add_argument('-d', "--noDebug", action='store_true', help='filter out debug messages')
```

Comment:
maybe change it to 
{code}
parser.add_argument('-D', "--debug", action='store_false', help='show debug messages')
{code}